addons/web/static/src/css/base.sass

```diff
@@ -2124,9 +2124,10 @@ $sheet-padding: 16px
             padding: 0 !important
             cursor: ns-resize
             .oe_list_handle
-                font-size: 18px
+                font-size: 7px
                 color: #E0E0E0
-                margin-right: 7px
+                margin-right: 10px
+                margin-top: 8px
```

Comment:
@jem-odoo actually i don't understand your point. :disappointed: can you explain?